src/main/java/com/e_gineering/maven/gitflowhelper/TagMasterMojo.java

```diff
@@ -47,6 +47,9 @@ protected void execute(final GitBranchType type, final String gitBranch, final S
                 gitURLExpression = "${env.GIT_URL}";
             }
             String gitURL = resolveExpression(gitURLExpression);
+            if (!gitURL.startsWith("scm:git:")) {
```

Comment:
I did some testing with `scm:validate` and both the `scm:` prefix and the provider are case-sensitive and must be lowercase:

```
[ERROR] Validation of scm url connection (project.scm.developerConnection) failed :
[ERROR] The scm url must start with 'scm:'.
[ERROR] No such provider installed 'GIT'.
[ERROR] The invalid scm url connection: 'SCM:GIT:ssh://git@xxx/yyy/zzz.git'.
```

So I guess it should be safe to compare with lower case only.